modules/ROOT/pages/evaluate-dw-expressions.adoc

```diff
@@ -0,0 +1,57 @@
+= DataWeave Expression Watches
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+When debugging your Mule application, you may need to evaluate a specific expression while your flow's execution is paused. You can write a DataWeave expression to evaluate or inspect any message property.
+
+If your expression is a declaration, the debugger returns `boolean`. Otherwise, it returns the value.
+For example, you can set:
+
+* One expression to evaluate if the method for a request is `GET`:
++
+[source]
+--
+#[attributes.method == "GET"]
+--
+* And another expression to inspect the payload of the message:
++
+[source]
+--
+#[payload]
+--
+
+The first expression returns either `true` or `false`, while the second expression displays the value of the payload at that particular breakpoint.
+
+You can also save these evaluations as expression watches so that the debugger displays the result of these evaluations every time the execution of your flow stops at a breakpoint.
```

Comment:
So, you write an expression, the expression evaluates any message property at a breakpoint and displays it, and you can save the result as "an expression watch"?

Sorry I am so unfamiliar with this language. If your audience will not be confused by its use or already knows the definitions, ignore the edit questions. But please consider defining each term and its relation to the others:

expression
expression watch
watch
evaluation
what user does vs what debugger does
order of actions